### PR TITLE
Added support for specifying arima method

### PR DESCRIPTION
**Pull request by mitchelloharawild**
Implements #741

### CODE REVIEW

File: R/newarima2.R [hunk 1]
```diff
@@ -711,10 +713,12 @@ myarima <- function(x, order = c(0, 0, 0), seasonal = c(0, 0, 0), constant=TRUE,
   m <- frequency(x)
   use.season <- (sum(seasonal) > 0) & m > 0
   diffs <- order[2] + seasonal[2]
-  if (approximation) {
-    method <- "CSS"
-  } else {
-    method <- "CSS-ML"
+  if(is.null(method)){
```
Comment (robjhyndman) [inline R/newarima2.R]: The default setting will have method="CSS-ML", so this will not be triggered.

Comment (mitchelloharawild) [reply R/newarima2.R]: Great, I thought there was something I had missed!

File: R/newarima2.R [hunk 2]
```diff
@@ -146,6 +147,7 @@ auto.arima <- function(y, d=NA, D=NA, max.p=5, max.q=5,
   }
   ic <- match.arg(ic)
   test <- match.arg(test)
+  method <- match.arg(method)
```
Comment (robjhyndman) [inline R/newarima2.R]: Need to remove this match.arg now.

Comment (mitchelloharawild) [reply R/newarima2.R]: Oops. Fixed.

**Pull request merged**